client/web/src/enterprise/codeintel/shared/components/CodeIntelUploadOrIndexIndexer.tsx

```diff
@@ -2,10 +2,12 @@ import React, { FunctionComponent } from 'react'
 
 import { LsifIndexFields } from '../../../../graphql-operations'
 
+import { CodeIntelIndexer } from './CodeIntelIndexer'
+
 export interface CodeIntelUploadOrIndexIndexerProps {
-    node: Partial<Pick<LsifIndexFields, 'inputIndexer'>>
+    node: Partial<Pick<LsifIndexFields, 'indexer'>>
 }
 
 export const CodeIntelUploadOrIndexIndexer: FunctionComponent<CodeIntelUploadOrIndexIndexerProps> = ({ node }) => (
-    <span>{node.inputIndexer}</span>
+    <span>{node.indexer && <CodeIntelIndexer indexer={node.indexer} />}</span>
```

Comment:
do you want to render an empty span when no indexer is present? Otherwise I would early return `null` before.
```suggestion
   if (!node.indexer) { return null }
   return <span><CodeIntelIndexer indexer={node.indexer} /></span>
```
.. and looking at this.. is the span even required or did you want to have an empty fragment `<>`?